cmd/immudb/immudb.go

```diff
@@ -27,6 +27,12 @@ import (
 	"github.com/spf13/viper"
 )
 
+var App = "immudb"
+var Version = "v0.0.1"
```

Comment:
Why we need to populate the var version? It should arrives in the compiling time. Shouldn't?